DOCUMENTATION.md

```diff
@@ -515,7 +516,30 @@ You must then call either the `.links` method or the `.collection` method on thi
 |`response.links`|Returns a hash linking to the requested notifications (limited to 250)|
 |`response.collection`|Returns an array of the required notifications|
 
-If you call the `collection` method on this object to return an array, you will then have to call different methods on this array to return the requested information.
+If you call the `collection` method on this object to return an array of notifications, you must then call the following methods on this array to return information on individual notifications:
```

Comment:
Just a couple of really, minor wording tweaks - should be 'on the notifications in this array' instead of 'on this array'